### ISSUE TITLE
Error on installation when using python 3.8 on a Mac

### ISSUE THREAD

**Issue by Mineinjava** [gunthercox/ChatterBot#2051]
when I install chatterbot with `python3 -m pip install chatterbot`, I get this error
I am running python 3.8 on a Mac.

https://ghostbin.co/paste/55m2

**Comment by AnAnalogGuy** [comment]
I got the same. You either need to use python 3.7 or an earlier version of chatterbot.

**Comment by mkkeffeler** [comment]
yep, well said @AnAnalogGuy its due to the spacy 2.1.9 dependency, which needs to be upgraded with https://github.com/gunthercox/ChatterBot/pull/1963 being merged. please comment so we can get it done!
